docs/pages/guides/datasets.mdx

```diff
@@ -74,139 +73,73 @@ Any dataset that is referenced by a query will automatically be added to that qu
-  in back-ticks: <code>`template_string`</code>
-</div>
-
-<br />
-
-Multiple arguments can be provided to `postOps()` and `preOps()`, each a seperate query, which will be executed in the provided order. You can alternatively use a multi-line configuration statement:
-
-```js
-/*js
-```
-
-<div style={{ marginTop: "-40px" }} />
-
-```js
-postOps([
-  `GRANT SELECT ON ${self()} TO GROUP "allusers@dataform.co"`,
-  `GRANT SELECT ON ${self()} TO GROUP "otherusers@dataform.co"`
-]);
-```
-
-<div style={{ marginTop: "-40px" }} />
-
-```js
-*/
-```
-
-## Overriding the output schema
-
-By default, all datasets are created within the default schema chosen when initializing a project, usually `dataform`.
-
-This can be overridden on a per-file basis by changing the file name to have a fully qualified dataset name.
-For example, to create a dataset called `example` in the schema `other_schema`, create a file with the name `definitions/other_schema.example.sql`.
-This will override the output schema of the dataset to `other_schema`. To reference datasets with overridden schemas, use the fully qualified name of the dataset:
-
-```js
-SELECT * FROM ${ref("other_schema.table")}
-```
-
-## Configuration statements
-
-Each of the following examples of JavaScript configuration statements results in equivalent behavior.
-
-### Single line statements
-
-Every line starting with `--js` will be interpreted as JavaScript.
-
-```js
---js type("table");
---js disabled(true);
-```
-
-### Multi-line statements
 
-To write JavaScript over multiple lines, create a comment block that starts with `/*js` and ends with `*/`:
-
-```js
-/*js
+post_operations {
+  GRANT SELECT ON ${self()} TO GROUP "allusers@dataform.co"
+}
 ```
 
-<div style={{ marginTop: "-40px" }} />
+To specify multiple operations, separate them with `---`:
 
 ```js
-type("table");
-disabled(true);
+post_operations {
+  GRANT SELECT ON ${self()} TO GROUP "allusers@dataform.co"
+  ---
+  GRANT SELECT ON ${self()} TO GROUP "otherusers@dataform.co"
+}
 ```
 
-<div style={{ marginTop: "-40px" }} />
-
-```js
-*/
-```
+## Overriding a dataset's schema or name
 
-### Setting multiple properties with `config()`
-
-Most settings for a dataset can be passed in a single JSON object using the `config()` method:
-
-```js
-/*js
-```
-
-<div style={{ marginTop: "-40px" }} />
+By default, a dataset's schema is set to the default schema chosen when initializing a project, usually `dataform`, and a dataset's name is set to the name of the corresponding file.
+This can be overridden on a per-file basis by specifying the `schema` and/or `name` options. For example, to create a dataset called `example` in the schema `other_schema`:
 
 ```js
-config({
-  type: "table",
-  disabled: true
-});
+config {
+  schema: "other_schema",
+  name: "example"
+}
+SELECT * FROM ...
 ```
 
-<div style={{ marginTop: "-40px" }} />
+To reference datasets with overridden schemas, use the fully qualified name of the dataset:
```

Comment:
We probably should make a point that even when just changing the name, you need to use that new name as the ref value rather than the name of the file.

Reply:
good call, done